specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/IncreaseLineOfCreditBySubscription.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview",
+    "subscriptionId": "{subscriptionId}",
+    "parameters": {
+      "desiredCreditLimit": 10000
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingAccounts/default/lineOfCredit/default",
+        "name": "default",
+        "type": "Microsoft.Billing/lineOfCredit",
+        "properties": {
+          "creditLimit": {
+            "currency": "USD",
+            "value": 20000.00
+          },
+          "reason": null,
+          "remainingBalance": {
+            "currency": "USD",
+            "value": 6000.00
+          },
+          "status": "approved"
+        }
+      }
+    },
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com/providers/Microsoft.Billing/operationResults/44000000-0000-0000-0000-000000000000?api-version=2018-11-01-preview",
```

Comment:
@asarkar84 any reason you are sending location as well as Azure-AsyncOperation header.
Why not just use one. I know that it's allowed, would like to know for relying on location header protocol.

Reply:
That is true, Azure-AsyncOperation is the recommended way to poll the async operation. 
Once the Azure-AsyncOperation operation is completed that will return location in response. Some of our consumers use location property so we keep both. 
We follow the same pattern in our existing APIs for 202 status.